static/app/components/events/featureFlags/featureFlagDrawer.tsx

```diff
@@ -122,23 +180,39 @@ export function FeatureFlagDrawer({
         </InputGroup.TrailingItems>
       </InputGroup>
       <CompactSelect
+        value={sortGroup}
+        options={SORT_GROUP_OPTIONS}
+        triggerProps={{
+          'aria-label': t('Sort Group'),
+        }}
+        onChange={selection => {
+          setFlagSort(getDefaultFlagSort(selection.value));
+          setSortGroup(selection.value);
+        }}
+        trigger={triggerProps => (
+          <DropdownButton {...triggerProps} size="xs">
+            {getSortGroupLabel(sortGroup)}
+          </DropdownButton>
+        )}
```

Comment:
It seems like `triggerProps` above and `size="xs"` below could be done in the same way, either both up or both down in the jsx.

Reply:
oo, i'll just combine them all into the dropdown